cluster-autoscaler/cloudprovider/azure/test/Makefile

```diff
@@ -46,8 +46,15 @@ install-e2e: $(HELM)
 		--set image.repository=$(IMAGE)-$(GOARCH) \
 		--set image.tag=$(TAG) \
 		--set image.pullPolicy=Always \
+		--set extraArgs.scale-down-delay-after-add=10s \
```

Comment:
Basically what I was aiming for with this new config was "react as fast as possible" so any feedback on how to best achieve that is most welcome.

Reply:
sgtm